src/locale/locales/de/messages.po

```diff
@@ -1757,20 +1478,20 @@ msgstr "Dunkles Thema"
 
 #: src/screens/Signup/StepInfo/index.tsx:191
 msgid "Date of birth"
-msgstr ""
+msgstr "Geburtsdatum"
 
 #: src/screens/Settings/components/DeactivateAccountDialog.tsx:73
 #: src/view/screens/Settings/index.tsx:807
 msgid "Deactivate account"
-msgstr ""
+msgstr "Konto deaktivieren"
 
 #: src/view/screens/Settings/index.tsx:819
 msgid "Deactivate my account"
-msgstr ""
+msgstr "Deaktiviere mein Konto"
```

Comment:
```suggestion
msgstr "Mein Konto deaktivieren"
```

`Deaktiviere mein Konto` means `YOU must deactivate my account`, `Mein Konto deaktivieren` means `I deactivate my account`